src/test/java/homework1/AccountServiceTest.java

```diff
@@ -0,0 +1,147 @@
+package homework1;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.time.Year;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AccountServiceTest {
+    AccountService accountService = new AccountService();
```

Comment:
usually assignment of classes put in setup() method with Before/BeforeAll annotation